app/models/project.js

```diff
@@ -298,15 +298,6 @@ export default class extends Model {
     return wizard(projectProcedure, this);
   }
 
-  @computed('currentStep')
-  get currentStepNumber() {
-    const currentStep = this.get('currentStep');
-    if (currentStep.step === 'rezoning') { return 3; }
-    if (currentStep.step === 'complete') { return 3; }
-    if (currentStep.step === 'project-area') { return 2; }
-    return 1;
-  }
```

Comment:
I think actually this could be refactored to stay in the model since it is computed downstream of model information. 

Sequence is implied at https://github.com/NYCPlanning/labs-applicantmaps/pull/325/files#diff-d1ace59c7fda942c691a37cd9ad5d941R59. You could probably use [findIndex](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex) to determine the implicit order of a step:

```javascript
  @computed('currentStep')
  get currentStepNumber() {
    const { step: currentStepName } = this.get('currentStep');
	return projectProcedure.findIndex(({ step }) => {
		return step === currentStepName;
    });
  }
```

What do you think?

Reply:
Wouldn't this way still have the same problem outlined in #292? 

> …updating at the wrong time. If you say "no" to the Project Area question, the step changes to Rezoning before you click "Next –>".